src/app/components/query-explainer/query-explainer-routing.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from "@angular/core";
+import { RouterModule, Routes } from "@angular/router";
+import { QueryExplainerComponent } from "./query-explainer.component";
+
+const routes: Routes = [
+    {
+        path: "",
+        component: QueryExplainerComponent,
+    },
+];
+
+@NgModule({
+    imports: [RouterModule.forChild(routes)],
```

Comment:
Not sure I understand why we need another routing level, the paths are de-facto empty, there is just 1 component.

Reply:
Yes, you are right, I removed the extra routing.